prepare-tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# This script can be used to warmup the environment and execute the tests
+# It is used by the docker image at startup
+
+cp pom.xml ./tests/pom.xml
```

Comment:
Is there a particular reason for having both `prepare-tests.sh` and `env.build.sh` ?

I feel you can probably remove `env.build.sh` having any of its content merged when applicable

Reply:
I forgot about the env.build.sh script and I'll update it and remove the prepare-tests script